internal/lookout/ui/src/services/lookoutV2/JobsTablePreferencesService.ts

```diff
@@ -62,7 +62,7 @@ type QueryStringJobFilter = {
 // Keys are shortened to keep URL size lower
 export interface QueryStringPrefs {
   // Grouped columns
-  g: string[] | [null]
+  g: string[]
```

Comment:
This is a drive-by change, because I noticed that we never produce `[null]` here anymore (see 85fdace5d616c83e42ef9853744b5c6bc92ba6c0).